### PR TITLE
Add JSONLines accept type

### PR DESCRIPTION
**Pull request by andremoeller**
*Issue #, if available:*

*Description of changes:*

If "application/jsonlines" is the Accept type, remove newlines. This is so that MultiRecord Batch can join these outputs by newlines and make a parseable JSONLines file.

Apache bench was within noise, for small payloads and for large payloads.

Also, I can't get local testing (with tox or pytest) to work without changing code -- is there more to it than building the images and running tox?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/integration/local/test_container.py [hunk 1]
```diff
@@ -240,3 +240,17 @@ def test_predict_no_custom_attributes_header():
...
+    x = {
+        'instances': [1.0, 2.0, 5.0]
+    }
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept':  'application/jsonlines'
+    }
+    response = requests.post(BASE_URL, data=json.dumps(x), headers=headers)
+    y = json.loads(response.content.decode('utf-8'))
```
Comment (jesterhazy) [inline test/integration/local/test_container.py]: we need another assert here on the raw string response, to ensure it's really on one line and not just valid multi-line json.

File: container/sagemaker/tensorflow-serving.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ function tfs_json_request(r, json) {
 
         if (accept == 'application/jsonlines') {
```
Comment (jesterhazy) [inline container/sagemaker/tensorflow-serving.js]:
```
if ('application/jsonlines' == accept || 'application/jsons' == accept)
```

File: container/sagemaker/tensorflow-serving.js [hunk 3]
```diff
@@ -81,6 +81,7 @@ function tfs_json_request(r, json) {
...
             body = body.replace(/\n/g, '')
+            reply.headers['Content-Type'] = 'application/jsonlines'
```
Comment (jesterhazy) [inline container/sagemaker/tensorflow-serving.js]:
```
reply.headers['Content-Type'] = accept
```

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=5fb39bb7-6bde-436e-8828-0b603fbea452%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2d72bef7-3fd6-4dfa-8154-3b9b7e590614%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by andremoeller** [comment]
Yikes, forgot to commit test change -- just updated.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://3r4crpxqvk.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=413748f6-2336-4473-82ce-c5c25552e76a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**